src/ui/units/workbooks/components/Table/WorkbookEntriesTable/Row/Row.scss

```diff
@@ -0,0 +1,102 @@
+@import '~@gravity-ui/uikit/styles/mixins';
+
+.dl-content-row {
+    $class: &;
+
+    display: table-row;
+    color: inherit;
+    text-decoration: none;
+
+    &:hover {
+        background-color: var(--g-color-base-simple-hover-solid);
+        cursor: pointer;
+
+        #{$class}__btn-favorite {
+            visibility: visible;
+        }
+    }
+
+    &__empty-row {
+        display: table-row;
+    }
+
+    &__content-cell,
+    &__empty-cell {
+        display: table-cell;
+        box-sizing: content-box;
+        overflow: hidden;
+        overflow-wrap: break-word;
+        text-align: left;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        vertical-align: middle;
+
+        border-bottom: 1px solid var(--g-color-line-generic);
+    }
+
+    &__content-cell &__title-col,
+    &__content-cell &__control-col {
+        padding: 0 10px;
+    }
+
+    &__empty-cell {
+        color: var(--g-color-text-secondary);
+    }
+
+    &__title-col {
+        display: flex;
+        align-items: center;
+        max-width: 500px;
+
+        &_is-mobile {
+            max-width: 200px;
+        }
+    }
+
+    &__title-col-text {
+        @include overflow-ellipsis();
+        margin-left: 12px;
+    }
+
+    &__author-text {
+        overflow-wrap: break-word;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    &__control-col {
+        display: flex;
+        justify-content: flex-end;
+    }
+
+    &__btn-favorite {
+        color: var(--g-color-base-misc-heavy-hover);
+        width: 28px;
+        height: 28px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        border-radius: 4px;
+
+        &:hover {
+            background: var(--g-color-base-simple-hover);
+        }
+    }
+
+    &__btn-favorite {
+        visibility: hidden;
+
+        &_isFavorite {
+            visibility: visible;
+
+            #{$class}__icon-star-stroke {
+                visibility: hidden;
+            }
+        }
+    }
+
+    &__btn-favorite_isFavorite {
```

Comment:
1) duplicating the class above.
2) `isFavorite`-> `is-favorite`, naming classes in scss in `kebab-case`.

```scss
&__btn-favorite {
    visibility: hidden;

    &_is-favorite {
        visibility: visible;
    }
}
```